Javított bemutatkozás

megingat opened the pull request:
Kovács Ildikó vagyok. Kommunikáció szakon folytattam a tanulmányaimat. Mindig érdekelt az emberekkel való kapcsolattartás, a pszichológia. 2005-ben egy vezető ingatlanforgalmazó cégnél helyezkedtem el, majd ezt követően egyénileg folytattam a tevékenységemet. Az évek során magyar és külföldi partnerekkel együttműködve gyűjtöttem kapcsolatokat, tapasztalatokat, melyek a mai napig segítik munkámat. Vallom: "Válassz olyan foglalkozást, amit szeretsz, és soha többé nem kell igazán dolgoznod!"

Darabosné Zsóka vagyok. Pedagógiai, egészségügyi, szociálpedagógusi és coach végzettségeimmel feljogosítva testek, lelkek segítőjeként gyűjtöttem tapasztalatokat és kapcsolatokat, amiket hosszú évek óta kamatoztatok mint ingatlanközvetítő is. Ez a munka az egyik legizgalmasabb segítő tevékenység! :)

Két ember, két út, két élet... és egy varázslatos eladó ingatlannak köszönhetően ma munkatársak és barátok vagyunk... 
A Megingat Ingatlanközvetítő működésének ötletét a külön utakon szerzett, mégis hasonló tapasztalataink hívták életre.
Miként egy csipetnyi illatos fűszerrel megbolondított csokoládé, vagy egy meghitt olvasósarok a kicsi könyvesboltban, valahogy így szeretnénk az Ön különleges igényeinek megfelelni egy színes és színvonalas szolgáltatással. Önt akarjuk ahhoz segíteni, amit megálmodott, vagy amiről eddig talán álmodni sem mert. 
Ha Önnek azt mondták, hogy nem létezik, amit keres, kérjük, ne adja fel! Álma nem múlhat egy nem fellelt ingatlanon!  Beszéljünk arról, mi lenne Önnek az igazi, s mi megkeressük Önnek azt! :)
Köszönjük bizalmát és várjuk a hívását!
Ildikó és Zsóka

----bocs, nem tudtam csak a javítást küldeni, mert nem tudom, hol találom az eredetit... sajnos nem lettem okosabb. Van honnan fejlődnöm...

darabos commented:
Megtaláltam :). Szóval erről nem kaptam értesítést, mert ezt magadnak küldted.

> megingat  wants to merge 34 commits into `megingat:gh-pages` from `darabos:gh-pages`

A másik irányba kell küldened: a te repodból (`megingat:gh-pages`) az enyémbe (`darabos:gh-pages`).

Ha gondolod, betehetem a változtatásokat. Nem szívatni akarlak, csak érdemes megtanulnod, mert így később is könnyen megváltoztathatsz bármit.

megingat commented:
:) Köszönöm... még ne változtass, mert egy pici kiegészítést még meg kell
csinálnom.
Eszembe' sincs szívatásnak venni :)
Köszönöm, hogy tanítasz!
Az a baj, hogy nagyon hosszú volt a szünet, nagyon sok másra kellett
figyelnem, és a maradék agysejtjeim nem jól őrizték meg a számomra még
mindig kaotikus felületeket.
Egyszer meg kell rendesen jegyeznem!
Most ki is találtam a megoldást :)

Holnap találkozunk!!!  :D

Puszi

A.
1. március 12. 19:41 Daniel Darabos írta, notifications@github.com:

> Megtaláltam :). Szóval erről nem kaptam értesítést, mert ezt magadnak
> küldted.
> 
> megingat wants to merge 34 commits into megingat:gh-pages from
> darabos:gh-pages
> 
> A másik irányba kell küldened: a te repodból (megingat:gh-pages) az
> enyémbe (darabos:gh-pages).
> 
> Ha gondolod, betehetem a változtatásokat. Nem szívatni akarlak, csak
> érdemes megtanulnod, mert így később is könnyen megváltoztathatsz bármit.
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/megingat/megingat/pull/2#issuecomment-78562329.

darabos committed:
Merge pull request #115 from megingat/patch-106

Terület és ármódosítás!

megingat committed:
Árváltoztatás két helyen

Árváltoztatás két helyen...

darabos committed:
Merge pull request #117 from megingat/patch-108

Árváltoztatás két helyen

darabos committed:
Merge branch 'gh-pages' of https://github.com/megingat/megingat into megingat-gh-pages

darabos committed:
Merge branch 'megingat-gh-pages' into gh-pages

megingat committed:
Töröltem az árat!

Töröltem az árat, kiegészítettem a szöveget és egy képpel is bővítettem.

darabos committed:
Merge pull request #119 from megingat/patch-109

Töröltem az árat!

megingat committed:
Cím kiegészítése

Az ingatlan elkelt.

megingat committed:
110. sor kiegészítése

ELKELT az ingatlan. Ezt beírtam a címbe.

megingat committed:
Elütések javítása

Javítások, bővítés csak a szövegben.

megingat committed:
Javítások

Elütések javítása, ár módosítása.

megingat committed:
Elkelt

Cím kiegészítése.

megingat committed:
Sok módosítás

Sok változás miatt több helyen módosítottam a szöveget.

megingat committed:
Legfrissebb módosítás!!!

A tulajdonos kérésére további módosítások kerültek a szövegbe!

darabos committed:
Merge pull request #120 from megingat/patch-110

Cím kiegészítése

darabos committed:
Merge pull request #121 from megingat/patch-111

110. sor kiegészítése

darabos committed:
Merge pull request #122 from megingat/patch-112

Elütések javítása

darabos committed:
Merge pull request #123 from megingat/patch-113

Javítások

darabos committed:
Merge branch 'gh-pages' into patch-114

darabos committed:
Merge pull request #124 from megingat/patch-114

Elkelt

darabos committed:
Merge pull request #125 from megingat/patch-115

Sok módosítás

darabos committed:
Merge branch 'gh-pages' into patch-116

darabos committed:
Merge pull request #126 from megingat/patch-116

Legfrissebb módosítás!!!

megingat committed:
Javítás

Herend linkjének kitörlése.

megingat committed:
Elmaradt módosítások

Elmaradt javítások.

darabos committed:
Merge pull request #128 from megingat/patch-118

Elmaradt módosítások

megingat committed:
Árat módosítottam!

Árat módosítottam!

darabos committed:
Merge pull request #129 from megingat/patch-119

Árat módosítottam!

megingat committed:
Címet javítottam

Címet javítottam.

darabos committed:
Merge pull request #130 from megingat/patch-121

Címet javítottam

megingat committed:
Cím javítása

Még mindig ott volt a kettős kereszt a címkép előtt...

darabos committed:
Merge pull request #127 from megingat/patch-117

Javítás

darabos committed:
Merge pull request #131 from megingat/patch-122

Cím javítása

darabos committed:
Kell a "---".

darabos committed:
Limit image height.

darabos committed:
Sorry, I meant vh.

darabos committed:
Utolsó kép legyen nagy.

megingat committed:
Ár módosítása

Ár módosítása és javítások.

darabos committed:
Merge pull request #132 from megingat/patch-123

Ár módosítása

darabos committed:
Leányfalu listázása.

darabos committed:
Revert "Leányfalu elrejtése"

darabos committed:
Nem kell kétszer.

darabos committed:
Merge pull request #133 from darabos/revert-114-darabos-leanyfalu-le

Revert "Leányfalu elrejtése"

megingat committed:
Változott az ár

Változott az ár.

darabos committed:
Merge pull request #134 from megingat/patch-124

Változott az ár

megingat committed:
Módosítás

Terület méretének módosítása.

megingat committed:
Módosítás

Terület méretének módosítása.

megingat committed:
Módosítás

Terület méretének módosítása.

darabos committed:
Merge pull request #135 from megingat/patch-126

Módosítás

darabos committed:
Merge pull request #136 from megingat/patch-127

Módosítás

darabos committed:
Merge pull request #137 from megingat/patch-128

Módosítás

darabos committed:
Delete Ildikó from English footer too.

megingat committed:
Bővítettem, változtattam

Az árat módosítottam, az ingatlan jogi státuszát (remélem) pontosítottam.

megingat committed:
Módosítás a malomnál

A területet módosítottam a herendi malomnál.

darabos committed:
Merge pull request #139 from megingat/patch-130

Módosítás a malomnál

darabos committed:
Merge pull request #138 from megingat/patch-129

Bővítettem, változtattam

megingat committed:
Új hirdetés

Új hirdetést írtam. 
Biztosan kell még javítanom. :)

Köszönöm.

megingat committed:
Kiegészítés

Kőszegi ingatlan hirdetése.

megingat committed:
Ár módosítása

Árat módosítottam.

darabos committed:
Merge pull request #140 from megingat/patch-131

Új hirdetés

darabos committed:
Merge pull request #141 from megingat/patch-132

Kiegészítés

darabos committed:
Merge pull request #142 from megingat/patch-133

Ár módosítása

darabos committed:
Rename elado/Kőszeg/index.md to elado/koszeg/index.md

megingat committed:
Egy új hirdetés

Új hirdetés, új képek.

darabos committed:
Merge pull request #143 from megingat/patch-134

Egy új hirdetés

megingat committed:
Javítás

Beírtam egy szót és kivettem egy képet, mert az duplán volt.

darabos committed:
Merge pull request #145 from megingat/patch-137

Javítás

darabos committed:
Milyen sorkizártan?

darabos committed:
HTTPS for fonts.

darabos committed:
Convert Imgur links to HTTPS.

megingat committed:
Képet cseréltem

Találtam még egy duplikatúrát.

darabos committed:
Merge pull request #146 from megingat/patch-138

Képet cseréltem

megingat committed:
Többféle változtatás

8 képpel egészítettem ki, és változott az ár.

darabos committed:
Merge pull request #147 from megingat/patch-139

Többféle változtatás

megingat committed:
Árat módosítottam

Árat módosítottam.

darabos committed:
Merge pull request #148 from megingat/patch-140

Árat módosítottam

darabos committed:
Update footer.html

megingat committed:
Átírtam az árat

Átírtam az árat.

darabos committed:
Merge pull request #149 from megingat/patch-141

Átírtam az árat

megingat committed:
Árat módosítottam.

Árat módosítottam.

darabos committed:
Merge pull request #150 from megingat/patch-142

Árat módosítottam.

megingat committed:
Változtatás a szövegen

Az ingatlan jelenleg nem eladó. Ennek megfelelően változott a szöveg.

megingat committed:
Megváltozott helyzet miatt megváltozott szöveg

Megváltozott helyzet miatt megváltozott szöveg.

megingat committed:
Halgazdaság és Majk nem eladó

Halgazdaság és Majk már nem eladó.

megingat committed:
Képek hozzáadésa

Két új fotóval bővítettem.

darabos committed:
Merge pull request #151 from megingat/patch-143

Változtatás a szövegen

darabos committed:
Merge pull request #152 from megingat/patch-144

Megváltozott helyzet miatt megváltozott szöveg

darabos committed:
Merge pull request #153 from megingat/patch-145

Halgazdaság és Majk nem eladó

darabos committed:
Merge pull request #154 from megingat/patch-146

Képek hozzáadása

megingat committed:
Ármódosítás

Árat, címet változtattam.

megingat committed:
Címmódosítás

Címmódosítás Leányfalunál.

darabos committed:
Merge pull request #155 from megingat/patch-147

Ármódosítás

darabos committed:
Merge pull request #156 from megingat/patch-148

Címmódosítás

megingat committed:
Több módosítás

Árat módosítottam, szöveget bővítettem, 4 képet adtam hozzá.

darabos committed:
Merge pull request #157 from megingat/patch-149

Több módosítás

megingat committed:
Terület és ár változott

Terület pontosítása, ár emelése.

megingat committed:
Terület változott

Malomnál a terület változott.

darabos committed:
Merge pull request #158 from megingat/patch-150

Terület és ár változott

darabos committed:
Merge pull request #159 from megingat/patch-151

Terület változott

megingat committed:
Árat változtattak

A legfrissebb árváltoztatás.

darabos committed:
Merge pull request #160 from megingat/patch-152

Árat változtattak

megingat committed:
Kiegészítés

Két ingatlannal kiegészítve

megingat committed:
Őrséggel kiegészítve

Őrséggel kiegészítve.

darabos committed:
Merge pull request #161 from megingat/patch-153

Kiegészítés

darabos committed:
Merge pull request #162 from megingat/patch-154

Őrséggel kiegészítve

darabos committed:
Kisebb képek hogy gyorsabban töltődjön az oldal.

darabos committed:
Merge pull request #163 from darabos/darabos-kisebb-kepek

Kisebb képek hogy gyorsabban töltődjön az oldal

darabos committed:
Jobban olvasható feliratok fehér háttéren.

megingat committed:
A mai módosítás

A tulajdonos igényei...

darabos committed:
Merge pull request #167 from megingat/patch-158

A mai módosítás

megingat committed:
Egy képpel kevesebb...

A tulajdonos mai kérésére levettem az alaprajzot.

darabos committed:
Merge pull request #168 from megingat/patch-159

Egy képpel kevesebb...

megingat committed:
Ármódosítás

Már megint módosították az árat.

darabos committed:
Merge pull request #169 from megingat/patch-160

Ármódosítás

megingat committed:
Új hirdetés

Nadapi hirdetés

megingat committed:
Új hirdetés

Új hirdetés.

megingat committed:
Új hirdetés

Nadap.

darabos committed:
Merge pull request #171 from megingat/patch-162

Új hirdetés

darabos committed:
Merge pull request #172 from megingat/patch-163

Új hirdetés

darabos committed:
Merge pull request #170 from megingat/patch-161

Új hirdetés

darabos committed:
#170 javítása.

darabos committed:
Merge pull request #173 from darabos/darabos-patch-1

Címek összekeveredtek

megingat committed:
Pontosítás

Remélem, így érthetőbb.

megingat committed:
Még egy vesszőt is kivettem

Remélem, így érthetőbb.

darabos committed:
Merge pull request #174 from megingat/patch-164

Pontosítás

darabos committed:
Merge pull request #175 from megingat/patch-165

Még egy vesszőt is kivettem

megingat committed:
Mai módosítás, kiegészítés

Megint módosítottak az áron.

darabos committed:
Merge pull request #176 from megingat/patch-166

Mai módosítás, kiegészítés

darabos committed:
Antoinette villa törlése.

megingat committed:
Változtattam árat, szöveget

Változtattam árat, kiegészítettem a szöveget.

darabos committed:
Merge pull request #177 from megingat/patch-167

Változtattam árat, szöveget

darabos committed:
Malom ne legyen a listában.

darabos committed:
Merge pull request #178 from darabos/darabos-unlist-mill

Malom ne legyen a listában

megingat committed:
Ár

Ismét van irányár.

megingat committed:
Ár módosítása

Itt levettem a számértéket.

darabos committed:
Merge pull request #179 from megingat/patch-169

Ár

darabos committed:
Merge pull request #180 from megingat/patch-170

Ár módosítása

darabos committed:
Őrségi hirdetés törlése a listából.

darabos committed:
Merge pull request #181 from darabos/darabos-orsegtelen

Őrségi hirdetés törlése a listából

darabos committed:
Pannonhalmát kivettem a listából.

darabos committed:
Merge pull request #182 from darabos/darabos-pannonhalma-le

Pannonhalmát kivettem a listából

darabos committed:
Add "noindex" tags.

darabos committed:
Merge pull request #183 from darabos/darabos-canindex

Add "noindex" tags

darabos committed:
"no" is parsed as false.

megingat committed:
Sok változtatás

Emelkedett az ár, ezzel összefüggő javítások a szövegben...

darabos committed:
Merge pull request #184 from megingat/patch-171

Sok változtatás

darabos committed:
Leányfalu deleted.

darabos committed:
Delete Leanyfalu

darabos committed:
Delete Leanyfalu

darabos committed:
Delete Leanyfalu

darabos committed:
Merge pull request #185 from darabos/darabos-leanyfalu-delisting

Leányfalu törlése

megingat committed:
Ár megegyezés szerint

Ár megegyezés szerint - átmenetileg.

darabos committed:
Merge pull request #186 from megingat/patch-172

Ár megegyezés szerint

megingat committed:
MEGSZÜLETETT AZ ÁR.

ELFOGADTA A JAVASLATOMAT...

darabos committed:
Merge pull request #187 from megingat/patch-173

MEGSZÜLETETT AZ ÁR.

megingat committed:
Ármódosítás

Árat változtattam.

megingat committed:
Árváltozás!

Emelkedett az ára.

darabos committed:
Merge pull request #189 from megingat/patch-176

Árváltozás!

darabos committed:
Merge pull request #188 from megingat/patch-175

Ármódosítás

megingat committed:
Megosztva is eladó lett

Sok helyen módosítottam.

darabos committed:
Merge pull request #190 from megingat/patch-177

Megosztva is eladó lett

megingat committed:
Változott az ár

Változott az ár.

darabos committed:
Merge pull request #191 from megingat/patch-178

Változott az ár

megingat committed:
Árváltozás

Változott az ár. Megint.

darabos committed:
Merge pull request #192 from megingat/patch-179

Árváltozás

megingat committed:
Töröltem az összeghatárt

Mindkét helyen töröltem az összeghatárt.

megingat committed:
Újra írtam irányárat

Újra írtam irányárat.

darabos committed:
Merge pull request #195 from megingat/patch-182

Újra írtam irányárat

darabos committed:
Revert "Malom ne legyen a listában."

This reverts commit b4ab801f03d372ae426f1feb9bbf34f264d30567.

darabos committed:
Merge pull request #196 from darabos/darabos-ujra-malom

Malom vissza a listába

darabos committed:
Nem "mindketten".

darabos committed:
Gomb a keresett ingatlanokhoz.

darabos committed:
Merge pull request #197 from darabos/darabos-fixes

Kis javítások

darabos committed:
Kőszeg törölve.

darabos committed:
Merge pull request #198 from darabos/darabos-no-koszeg

Kőszeg törölve

megingat committed:
Képekkel bővítettem

Képekkel bővítettem, és kicsit módosítottam a szöveget.

darabos committed:
Merge pull request #199 from megingat/patch-183

Képekkel bővítettem

megingat committed:
Create index.md

Írtam egy új hirdetést. 
Félek, hogy elveszik a munkám, mert nem látom át, mit tegyek most. :(
Jó, ha ezt neked elküldöm? Egyáltalán ez elmegy neked?
Nem tudom, mivel tudnám menteni! 
Már várják, hogy megjelenjen. 
Ha sikerül, csinálom a szentendreit is.
Köszönöm szépen! :)

darabos committed:
Merge pull request #200 from megingat/patch-186

Create index.md

megingat committed:
Create index.md

Az új, szentendrei hirdetés.
Remélem, ez is sikerül...  :)

megingat committed:
Update index.md

Kiegészítettem a két új hirdetéssel!

darabos committed:
Merge pull request #201 from megingat/patch-187

Create index.md

darabos committed:
Merge pull request #202 from megingat/patch-188

Update index.md

darabos committed:
Merge pull request #193 from megingat/patch-180

Töröltem az összeghatárt

megingat committed:
Update index.md

Remélem, sikerült!
Köszönöm szépen!!!! 
:)

darabos committed:
Még egy </div>

darabos committed:
Merge pull request #203 from megingat/patch-189

Update index.md

megingat committed:
Update index.md

Árat csökkentettem.

megingat committed:
Update index.md

Méretet, árat is módosítottam.

darabos committed:
Merge pull request #204 from megingat/patch-190

Update index.md

darabos committed:
Merge pull request #205 from megingat/patch-191

Update index.md

megingat committed:
Update index.md

Címen, áron változtattam.

darabos committed:
Merge pull request #206 from megingat/patch-192

Update index.md

megingat committed:
Update index.md

Hibákat javítottam.

megingat committed:
Update index.md

Itt is címet javítottam.

darabos committed:
Merge pull request #207 from megingat/patch-193

Update index.md

darabos committed:
Merge pull request #208 from megingat/patch-194

Update index.md

megingat committed:
Update index.md

Árváltozás, kiegészítés, képek

darabos committed:
Merge pull request #209 from megingat/patch-195

Update index.md

megingat committed:
Szöveg pontosítása

Adatok pontosítása a szövegben.

darabos committed:
Merge pull request #210 from megingat/patch-196

Szöveg pontosítása

megingat committed:
Update index.md

A szöveg sok helyen változott.

darabos committed:
Merge pull request #211 from megingat/patch-197

Update index.md

megingat committed:
Update index.md

Kis módosítás a szövegben.

megingat committed:
Update index.md

Címoldalon módosítottam.

darabos committed:
Merge pull request #213 from megingat/patch-199

Update index.md

darabos committed:
Merge pull request #212 from megingat/patch-198

Update index.md

darabos committed:
Nadap törlése.

darabos committed:
Nadap törlése.

darabos committed:
Nadap törlése.

darabos committed:
Merge pull request #214 from darabos/darabos-nadap-torles

Nadap törlése

darabos committed:
Balatonfüred törlése a listából.

darabos committed:
Merge pull request #215 from darabos/darabos-no-balatonfured

Balatonfüred törlése a listából

megingat committed:
Update index.md

Feltettem a képeket!

darabos committed:
Merge pull request #216 from megingat/patch-200

Update index.md

megingat committed:
Update index.md

Ár pontosítása.

darabos committed:
Merge pull request #217 from megingat/patch-201

Update index.md

megingat committed:
Update index.md

A tulaj meggondolta magát. A birtok nem eladó. 
Az eladással kapcsolatban mindent múlt időbe tettem.
Megjelentetni nem kell, csak megtalálták valahol, és  fennakadtak...

darabos committed:
Merge pull request #218 from megingat/patch-766654

Update index.md

darabos committed:
Delete elado/orseg/index.md

darabos committed:
Merge pull request #219 from darabos/darabos-patch-1

Delete elado/orseg/index.md

megingat committed:
Frissített ár

darabos committed:
Merge pull request #220 from megingat/patch-223181

Frissített ár

megingat committed:
Terület változott

darabos committed:
Merge pull request #221 from megingat/patch-373860

Terület változott

megingat committed:
Több változás

Ár és terület változott.

darabos committed:
Merge pull request #222 from megingat/patch-613044

Több változás

megingat committed:
Update index.md

Kivettem egy képet, mert lebontották azt a viskót. 
Köszönöm!!!

darabos committed:
Merge pull request #223 from megingat/patch-949229

Update index.md

darabos committed:
Delete for-sale/leanyfalu/index.md

darabos committed:
Delete elado/leanyfalu/index.md

megingat committed:
Átmenetileg nem eladó

Átírtam az "eladó"-t és az árat.

darabos committed:
Merge pull request #224 from megingat/patch-811117

Átmenetileg nem eladó

megingat committed:
Képeket cseréltem

Képeket cseréltem, az ár már régebben változott.
Nem értem, itt miért jó, ha a megjelenő oldalon még mindig a régi. 
Nem tudom, mi a baj a 18. és 38. sorral! Nem én írtam az "éves"-t, nem tudom javítani!

darabos committed:
Merge pull request #225 from megingat/patch-819427

Képeket cseréltem

darabos committed:
Update CNAME

megingat committed:
Update index.md

Árat módosítottam.

megingat committed:
Árat módosítottam

Változott az ár!

megingat committed:
Update index.md

Képeket cseréltem, adtam hozzá.

darabos committed:
Merge pull request #226 from megingat/patch-968390

Update index.md

darabos committed:
Merge pull request #227 from megingat/patch-263993

Árat módosítottam

darabos committed:
Merge pull request #228 from megingat/patch-362297

Update index.md

megingat committed:
Árat módosítottam

Árat módosítottam, a  valamiért megváltozott képet inkább töröltem.

darabos committed:
Merge pull request #230 from megingat/patch-132103

Árat módosítottam

megingat committed:
Update index.md

Megint ármódosítás!

megingat committed:
Update index.md

A malom területének módosítása.

darabos committed:
Merge pull request #231 from megingat/patch-234983

Update index.md

darabos committed:
Merge pull request #232 from megingat/patch-210366

Update index.md

megingat committed:
Update index.md

Árat módosítottam, egy linket töröltem.

darabos committed:
Merge pull request #233 from megingat/patch-640231

Árat módosítottam

megingat committed:
Árváltozás

Ár módosítása

darabos committed:
Merge pull request #234 from megingat/patch-378066

Árváltozás
